File: LENTABP10/src/main/java/com/lenta/lentabp10/models/WriteOffReasonRestInfo.java
package com.lenta.lentabp10.models;

/// <summary>
/// Мат номер, элемент списка IT_MATERIALS в SaveTaskDataToSapRestRequest
/// </summary>

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

public class WriteOffReasonRestInfo {

    /// Номер товара
    @JsonProperty("MATNR")
    @Getter private String matnr;

    /// Причина движения
    @JsonProperty("GRUND")
    @Getter private String writeOffCause;

    /// Место возникновения затрат
    @JsonProperty("KOSTL")
    @Getter private String kostl;

    /// Введенное количество
    @JsonProperty("FIRST_QNT")
    @Getter private String amount;

    public WriteOffReasonRestInfo(String matnr, String writeOffCause, String kostl, String amount) {
        this.matnr = matnr;
        this.writeOffCause = writeOffCause;
        this.kostl = kostl;
        this.amount = amount;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/models/WriteOffReasonRestInfo.kt
package com.lenta.bp10.models

import com.google.gson.annotations.SerializedName

class WriteOffReasonRestInfo(
        /// Номер товара
        @SerializedName("MATNR")
        val matnr: String,

        /// Причина движения
        @SerializedName("GRUND")
        val writeOffCause: String,

        /// Место возникновения затрат
        @SerializedName("KOSTL")
        val kostl: String,

        /// Введенное количество
        @SerializedName("FIRST_QNT")
        val amount: String
)